scripts/local.runner.js

```diff
@@ -2,15 +2,52 @@
 
 var Nightwatch = require('nightwatch');
 var browserstack = require('browserstack-local');
+var localConf = require('../conf/local.conf.js');
+
 var bs_local;
 
 try {
   process.mainModule.filename = "./node_modules/.bin/nightwatch"
 
   // Code to start browserstack local before start of test
-  console.log("Connecting local");
+  console.log('Connecting local');
   Nightwatch.bs_local = bs_local = new browserstack.Local();
-  bs_local.start({'key': process.env.BROWSERSTACK_ACCESS_KEY }, function(error) {
+
+  var localOptions = {
+    'key': process.env.BROWSERSTACK_ACCESS_KEY
```

Comment:
Can you change this to use `localConf` for the key ? It is present in the `test_settings` .